fdbserver/storageserver.actor.cpp

```diff
@@ -3890,12 +3977,99 @@ ACTOR Future<Void> serveGetKeyRequests( StorageServer* self, FutureStream<GetKey
 	}
 }
 
+ACTOR Future<Void> watchValueProxy( StorageServer* self, WatchValueRequest req, PromiseStream<WatchValueRequest> stream ) {
```

Comment:
I know that I came up with this name. But could we please try to find a better name for this proxy? I think the name is horrible and users will assume all kind of bad stuff when they see this show up in a trace or a sample